### DISCUSSION TITLE
ModuleNotFoundError: No module named 'ultralytics.data'

### DISCUSSION THREAD

**Discussion by lkw052715** [mikel-brostrom/yolo_tracking#1107] [Q&A] [answered]
ModuleNotFoundError: No module named 'ultralytics.data'How do I fix this? I use like'python track.py --yolo-model E:\pyproject\yolo_tracking-master\yolov8n.pt --source E:\pyproject\yolo_tracking-master\MOT17-09-SDP-raw.mp4'and'pip install ultralytics',but it didn't work

**Comment by mikel-brostrom** [comment]
You should not install the original ultrlytics package. We hold a custom ultralytics package.

**Reply by lkw052715** [reply]
Thanks for answering so quickly! Where can I find and install a custom ultralytics package, I have pip install -v -e .

**Reply by mikel-brostrom** [reply, marked as answer]
It gets installed during runtime if not installed. So just `pip uninstall ultralytics` and run the tracker.

**Reply by lkw052715** [reply]
Thank you very much for your help, I think all the configuration is ready, when I run the tracker, I get this error：requests.exceptions.ConnectionError: HTTPSConnectionPool(host='drive.google.com', port=443): Max retries exceeded with url: /uc?id=1sSwXSUlj4_tHZequ_iZ8w_Jh0VaRQMqF (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fe3f7859430>: Failed to establish a new connection: [Errno 110] Connection timed out'))
How do I fix this？

**Comment by mikel-brostrom** [comment]
If you are within China you need a VPN

**Reply by lkw052715** [reply]
Your suggestion works, but after I used the VPN, a new problem arose：requests.exceptions.SSLError: HTTPSConnectionPool(host='drive.google.com', port=443): Max retries exceeded with url: /uc?id=1sSwXSUlj4_tHZequ_iZ8w_Jh0VaRQMqF (Caused by SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1131)')))
Thanks again for your reply！

**Reply by lkw052715** [reply]
I used this to solve the above problem：pip install urllib3==1.25.11

**Comment by sidb236** [comment]
Hi, how do I access the custom ultralytics folder? I want to make some changes in my code. I will be grateful for your reply.

**Reply by sidb236** [reply]
I found it. 
Use the following command to find the path. As the recent modification has made the ultrlytics into a package, it can be found inside the Location: /usr/local/lib/python3.10/dist-packages in colab. 
!pip show ultralytics
